tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +157,15 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
```

Comment:
I think the case also need to run `setup_telemetry_forpyclient(duthost)` to setup telemetry on DUT, like the existing case `test_telemetry_ouput`?

Reply:
This is actually not required because the tests run sequentially. Once the first test runs `setup_telemetry_forpyclient` the subsequent once do not need to run. I also confirmed with @pra-moh.